setup.py

```diff
@@ -27,6 +28,13 @@ def get_version():
     assert False
 
 
+def get_long_description():
+    # type: () -> str
```

Comment:
Please add a blank line after the type hint.
I'm sorry that this is my mistake originally.